selftests/checkall

```diff
@@ -72,6 +89,7 @@ else
     [ ! $SELF_CHECK_CONTINUOUS ] && CMD+=" --failfast on"
     run_rc selftests "$CMD"
 fi
+[ "$AVOCADO_RESULTSDIR_CHECK" ] && run_rc job-results results_dir_content "$RESULTS_DIR_CONTENT"
```

Comment:
The `Temporary dirs` check is in Makefile, not in our `checkall` script. So we could move this to makefile, but I prefer keeping everything in one script rather than adding yet another Make target.